specification/cognitiveservices/Language.Conversations/main.tsp

```diff
@@ -162,23 +163,23 @@ model LuisParameters extends Record<unknown> {
 model LuisCallingOptions {
   @doc("Enable verbose response.")
   verbose?: boolean;
-
+  
```

Comment:
Nit: errant whitespace. Normally not a problem, but `git diff` will normally flag these.